crates/context/config/src/client/protocol/icp.rs

```diff
@@ -242,9 +242,15 @@ impl Network {
                     Ok(return_data) => Ok(return_data),
                     Err(err_msg) => Ok(err_msg.into_bytes()),
```

Comment:
here as well, but imo we shouldn't be decoding here

Reply:
Updated